code/modules/admin/verbs/debug.dm

```diff
@@ -499,3 +499,22 @@
 		for(var/atom/an_atom in world)
 			if(world.tick_usage > (100+(tick_offenses+jitter_this_run)))
 				stoplag()
+
+/**
+ * Enable Edge Dev Tools
+ */
+/client/proc/allow_browser_inspect()
```

Comment:
The proc itself should also contain a permissions check

Reply:
Scrolling up from the proc, there's various other procs (some I wrote, some I didn't) like this, that's because the only way to access the proc is by it being assigned to you, which seems sufficient (?)

Reply:
It *should* be sufficient but from a security perspective it is better to also check for proper permissions on the actual proc.